.github/ISSUE_TEMPLATE/release_candidate_checklist.md

```diff
@@ -0,0 +1,84 @@
+labels: ''
+assignees: ''
+
+---
+
+**Version**:  x.x.xrcx
+
+**Scheduled date**: XXX XX, 20XX
+
+**Before release**:
+
+- [ ] check if all tests pass on macOS, Linux and Windows
+- [ ] check if other GMT-derived projects work well
+	- [ ] MB-System (@PaulWessel)
+	- [ ] GMTSAR (@PaulWessel)
+	- [ ] GMT.jl (@joa-quim)
+	- [ ] PyGMT (@leouieda, @seisman, @weiji14)
+	- [ ] gmtmex (@PaulWessel, @joa-quim)
+- [ ] run `src/gmt_make_*.sh` to update some .c and .h files
+- [ ] run `admin/gs_check.sh` to test if latest ghostscript version works
+- [ ] update [changelog](/doc/rst/source/changes.rst)
+- [ ] update [INSTALL.md](/INSTALL.md) (only needed for major release candidates)
+- [ ] check if there are any warnings when building the documentation
+- [ ] add one new entry in `doc/rst/_static/version_switch.js` if it's a minor release
+- [ ] check/set values in `cmake/ConfigDefault.cmake`
+    - [ ] `GMT_VERSION_YEAR` is current year
+    - [ ] `GMT_PACKAGE_VERSION_*` is correctly set
+    - [ ] `GMT_LIB_SOVERSION` is correctly set
+    - [ ] set `GMT_PACKAGE_VERSION_SUFFIX` to rcx
+    - [ ] set `GMT_PUBLIC_RELEASE` to `TRUE`
+- [ ] freeze codes and commit all changes to GitHub
+
+**Release**:
+
+- [ ] create source tarballs (tar.gz and tar.xz) (@PaulWessel)
+- [ ] create macOS bundle (@PaulWessel)
+- [ ] create Windows installers (win64) (@joa-quim)
+- [ ] check if the source tarballs, macOS bundle and Windows installers work well
+- [ ] upload source tarballs, macOS bundle, Windows installers to the GMT FTP (@PaulWessel)
+- [ ] update README and VERSION files on the GMT FTP (@PaulWessel)
+- [ ] make a tag and push it to github (**Must be done after uploading packages to the GMT FTP**)
+    ```bash
+    # checkout master (for minor releases) or 6.x branch (for patch releases)
+    git checkout XXXX
+    # create the tag x.x.x
+    git tag x.x.x
+    # Push tags to GitHub
+    git push --tags
+    ```
+- [ ] make a GitHub release.
+  The GitHub Actions automatically create a draft release after pushing the tag to github.
+  We need to go to the [GitHub Release](https://github.com/GenericMappingTools/gmt/releases) page, and review it manually.
+  - [ ] 5 files are attached as release assets (2 source tarballs, 2 installers and 1 checksum file).
+  - [ ] download the checksum file and check if the checksums are correct
+  - [ ] edit the draft release, set the target to the correct tag, and publish the release
+- [ ] make announcements in the [GMT forum](https://forum.generic-mapping-tools.org/)
+- [ ] make announcements on the [GMT twitter](https://twitter.com/gmt_dev)
+- [ ] update links on the main site (News, Download & Documentation)
+
+**After release**:
+
+- [ ] create branch 6.x for bug-fixes if this is a minor release (i.e. create branch 6.1 after 6.1.0 is released)
+  ```
+  git checkout master
+  git checkout -b 6.1
+  git push --set-upstream origin 6.1
+  ```
+- [ ] update `GMT_PACKAGE_VERSION_*` in `cmake/ConfigDefault.cmake`
```

Comment:
No need to update `GMT_PACKAGE_VERSION_*`, right?

Reply:
So, after releasing 6.1.1 these were updated to 6.2.0. After releasing 6.2.0rc1 do we update the suffix to 6.2.0rc2 or leave it as 6.2.0rc1?

Reply:
I think we can just comment out the `set (GMT_PACKAGE_VERSION_SUFFIX "")` line, so that the version string is still like `6.2.0_eefd707_2021.04.06`, not `6.2.0rc1_eefd707_2021.04.06`.

Reply:
OK, done in https://github.com/GenericMappingTools/gmt/pull/5113/commits/fad5037e4d074eeaf62bab276fe9f64542384c39